packages/r-editor/package.json

```diff
@@ -36,15 +36,15 @@
   "dependencies": {
     "@elyra/script-editor": "3.7.0-dev",
     "@elyra/ui-components": "3.7.0-dev",
-    "@jupyterlab/application": "^3.0.0",
-    "@jupyterlab/apputils": "^3.0.0",
-    "@jupyterlab/codeeditor": "^3.0.0",
-    "@jupyterlab/docregistry": "^3.0.0",
-    "@jupyterlab/filebrowser": "^3.0.0",
-    "@jupyterlab/fileeditor": "^3.0.0",
-    "@jupyterlab/launcher": "^3.0.0",
-    "@jupyterlab/mainmenu": "^3.0.0",
-    "@jupyterlab/settingregistry": "^3.0.0",
+    "@jupyterlab/application": "^3.3.0-rc.0",
+    "@jupyterlab/apputils": "^3.3.0-rc.0",
+    "@jupyterlab/codeeditor": "^3.3.0-rc.0",
+    "@jupyterlab/docregistry": "^3.3.0-rc.0",
+    "@jupyterlab/filebrowser": "^3.3.0-rc.0",
+    "@jupyterlab/fileeditor": "^3.3.0-rc.0",
+    "@jupyterlab/launcher": "^3.3.0-rc.0",
+    "@jupyterlab/mainmenu": "^3.3.0-rc.0",
+    "@jupyterlab/settingregistry": "^3.3.0-rc.0",
     "@lumino/coreutils": "^1.5.3"
```

Comment:
coreutils needs to go to 1.5.6 to fix https://github.com/elyra-ai/elyra/security/dependabot/45